app/src/main/java/com/it/hungvt/movieapp/data/source/remote/api/service/MovieApi.java

```diff
@@ -0,0 +1,31 @@
+package com.it.hungvt.movieapp.data.source.remote.api.service;
+
+import com.it.hungvt.movieapp.data.model.Movie;
+import com.it.hungvt.movieapp.data.model.MovieResponse;
+import io.reactivex.Observable;
+import com.google.gson.JsonObject;
+import retrofit2.http.GET;
+import retrofit2.http.Path;
+import retrofit2.http.Query;
+
+public interface MovieApi {
+
+    @GET("movie/top_rated")
+    Observable<MovieResponse> getTopRate(@Query("api_key") String apiKey, @Query("page") int page);
```

Comment:
WARNING: [AvoidDuplicateLiterals] The String literal "api_key" appears 6 times in this file; the first occurrence is on line 14
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals